distribution/docker/ironbank-docker-build-context/build.gradle

```diff
@@ -3,6 +3,15 @@ import org.elasticsearch.gradle.DockerBase
 
 apply plugin: 'base'
 
+configurations {
+  transformLog4jJar
+}
+
+dependencies {
+  transformLog4jJar project(path: ":distribution:docker:transform-log4j-config", configuration: "default")
```

Comment:
if we are grabbing the default configuration then I don't think we need to specify `configuration: default` here. This can be simplified to just `transformLog4jJar project(':distribution:docker:transform-log4j-config')`

Reply:
👍

Reply:
Huh, turns out that if I remove `configuration: 'default'`, then the build fails. I've added it back in. 🤷‍♂️

Reply:
@breskeby any idea why?

Reply:
I should point out, it was the dependencies in the main docker build file where I had this problem, e.g. `distribution/docker/build.gradle`.